esbuild.config.js

```diff
@@ -13,11 +13,13 @@ const __dirname = path.dirname(fileURLToPath(import.meta.url));
 const kPublicDir = path.join(__dirname, "public");
 const kOutDir = path.join(__dirname, "dist");
 const kImagesDir = path.join(kPublicDir, "img");
+const kNodeModules = path.join(__dirname, "node_modules");
```

Comment:
```suggestion
const kNodeModulesDir = path.join(__dirname, "node_modules");
```

More accurate based on above constants, wdyt ?